app/Services/Contact/LifeEvent/CreateLifeEvent.php

```diff
@@ -72,15 +74,19 @@ public function execute(array $data) : LifeEvent
     private function addYearlyReminder($data, $lifeEvent)
     {
         if ($data['has_reminder']) {
-            $array = [
+            $date = Carbon::parse($data['happened_at']);
```

Comment:
You should use
```suggestion
            $date = DateHelper::parseDate($data['happened_at']);
```

Reply:
No because it's Y-m-d format and this helper needs `H:i:s` as well. It expects DateTime and I have a Date.

Reply:
Nope, `DateHelper::parseDateTime` parse a DateTime, but `DateHelper::parseDate` needs a Date format, and render a Date format.